src/components/SQForm/SQFormCheckboxGroup.js

```diff
@@ -80,13 +81,17 @@ function SQFormCheckboxGroup({
 
   return (
     <Grid item sm={size}>
-      <InputLabel id={groupLabel.toLowerCase()}>{groupLabel}</InputLabel>
-      <FormGroup row={shouldDisplayInRow}>
-        {childrenToCheckboxGroupItems()}
-      </FormGroup>
-      <FormHelperText required={isRequired}>
-        {HelperTextComponent}
-      </FormHelperText>
+      <FormControl component="fieldset">
+        <FormLabel component="legend" classes={{root: 'MuiInputLabel-root'}}>
+          {groupLabel}
+        </FormLabel>
+        <FormGroup row={shouldDisplayInRow}>
+          {childrenToCheckboxGroupItems()}
+        </FormGroup>
+        <FormHelperText required={isRequired}>
+          {HelperTextComponent}
+        </FormHelperText>
+      </FormControl>
```

Comment:
Changing `SQFormCheckboxGroup` and `SQFormRadioButtonGroup` to use `FormControl` and `FormLabel` instead of `InputLabel` will group the checkbox/radios together and allow us to query them by group, e.g., `screen.getByRole('group', { name: /field group name/i })`.

When writing tests this will help narrow down our queries when there are multiple inputs with the same name or value. 

A contrived example: a form may have three different radio groups where the options are 'yes' or 'no', with this structure we can confidently query an option within a specific group.
```js
// Narrowing it down to the group then picking the radio within that group
userEvent.click(
    within(screen.getByRole('group', { name: /group three/i })).getByRole('radio', { name: /yes/i })
)

// vs

// Querying all the radio buttons on the page by name then trying to find the right one within that array. 
userEvent.click(screen.getAllByRole('radio', { name: /yes/i })[2])
// The big issue here is that if another yes/no radio group is inserted in the dom ahead of the one we're trying to target,
// our intended target's place in the array will change and we'll now be clicking the wrong radio
```

This may not be all that useful when writing tests for these components, but if we expand testing to the other projects that use this library it should make writing tests for those much easier